src/app/app.component.ts

```diff
@@ -1,15 +1,17 @@
-import { Component } from '@angular/core';
-import { FormBuilder } from '@angular/forms';
+import { AfterViewInit, Component, ElementRef, ViewChild } from '@angular/core';
+import { FormBuilder, FormControl } from '@angular/forms';
 import { ExampleComponent } from './example/example.component';
 import { interval } from 'rxjs';
 import { finalize } from 'rxjs/operators';
+import { HelipopperDirective, HelipopperService } from '@ngneat/helipopper';
+import { MenuComponent } from './menu/menu.component';
```

Comment:
Why I can't see the menu component?

Reply:
sorry, some files were in another changelist, i committed them.